src/Sets/Ball2.jl

```diff
@@ -314,3 +315,34 @@ function chebyshev_center(B::Ball2{N}; compute_radius::Bool=false
     end
     return center(B)
 end
+
+"""
+    volume(B::Ball2{N}) where {N<:AbstractFloat}
+
+Return the volume of a ball in the 2-norm.
+
+### Input
+
+- `B` -- ball in the 2-norm
+
+### Output
+
+The volume of ``B``.
+
+### Algorithm
+
+This function implements the well-known formula for the volume of an n-dimensional
+ball using factorials. For details see the wikipedia article
+[Volume of an n-ball](https://en.wikipedia.org/wiki/Volume_of_an_n-ball).
```

Comment:
Maybe directly link to https://en.wikipedia.org/wiki/Volume_of_an_n-ball#Alternative_forms .